jwst/ramp_fitting/ramp_fit.py

```diff
@@ -1936,6 +2465,7 @@ def fit_lines(data, mask_2d, rn_sect, gain_sect, ngroups, weighting):
     else: # unsupported weighting type specified
         log.error('FATAL ERROR: unsupported weighting type specified.')
 
+    slope_s[good_pix] = slope
     slope_s[good_pix] = slope
```

Comment:
It looks to me like these 2 lines (2468-2469) are identical. Why the duplication?